devlab/tests/base.py

```diff
@@ -31,6 +32,36 @@
 OPENSTACK_RELEASES = {'192.168.1.2': 'grizzly',
                       '192.168.1.3': 'icehouse',
                       '192.168.1.8': 'juno'}
+SWIFT_AUTH_VERSION = '2'
+
+
+class SwiftConnection(object):
+    def __init__(self, auth_url, username, password, tenant):
+        self.auth_url = auth_url
+        self.username = username
+        self.password = password
+        self.tenant = tenant
+
+    def __enter__(self):
+        class Swift(object):
```

Comment:
At first, we cannot move initialization of swift connection to `__init__()` in `BasePrerequisites`, because we should close the connection at the end. This means that we should call `close()` method. Where can we do this in the `BasePrerequisites`? Because of this I decided that better way to do this with the `with` clause.
At second, let's imagine that someone could still instantiate `SwiftConnection` using its constructor without using the `with` clause. If we don't want that someone do this we should define the `Swift` class inside `__enter__()`.